cibuildwheel/linux.py

```diff
@@ -49,7 +49,7 @@ def get_python_configurations(
 
 def build(options: BuildOptions) -> None:
     try:
-        subprocess.check_output(['docker', '--version'])
+        subprocess.run(['docker', '--version'], check=True)
```

Comment:
```suggestion
        subprocess.run(['docker', '--version'], check=True, stdout=subprocess.DEVNULL)
```

This probably was to hide the output?

Reply:
Yes, that was the intention - we didn't care about the version, really it was just to check that docker was installed.

Reply:
perhaps adding a comment to that effect would be good, too
```suggestion
        # check docker is installed
        subprocess.run(['docker', '--version'], check=True, stdout=subprocess.DEVNULL)
```